src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs

```diff
@@ -67,14 +67,14 @@ private static void FirstChanceExceptionHandler(object sender, FirstChanceExcept
                 var assemblyDirectory = GetAssemblyDirectory();
                 var testName = CaptureTestNameAttribute.CurrentName ?? "Unknown";
                 var logDir = Path.Combine(assemblyDirectory, "xUnitResults", "Screenshots");
-                var baseFileName = $"{DateTime.Now:HH.mm.ss}-{testName}-{eventArgs.Exception.GetType().Name}";
+                var baseFileName = $"{DateTime.UtcNow:HH.mm.ss}-{testName}-{eventArgs.Exception.GetType().Name}";
```

Comment:
What's the impact of this? Just improves the experience of looking for a log for a particular integration test failure?

Reply:
We capture .NET and Watson event logs, and the timestamps in those files are UTC. Sometimes the events occurred during previous runs, so having the same dates for screenshots makes it clear if it's part of this run or a previous run.